tests/socket-select-win32.phpt

```diff
@@ -8,6 +8,7 @@ if (substr(PHP_OS, 0, 3) != 'WIN') {
 --FILE--
 <?php
     $socket = new Socket(\Socket::AF_INET, \Socket::SOCK_STREAM, \Socket::SOL_TCP);
+    $socket->bind("127.0.0.1", 19132);
```

Comment:
Not sure if 19132 is an acceptable port - I just automatically typed it in because this is the port I most commonly use in my project. If it should be changed, let me know.